src/test_scenes/scenePageView.js

```diff
@@ -59,8 +59,8 @@ class ScenePageView extends React.Component {
     ];
     return (
       <View position={this.props.position}>
-        <LinearLayout defaultItemAlignment={'top-center'} anchorPoint={'top-center'} orientation={'vertical'}>
-          <LinearLayout defaultItemPadding={[0.0, 0.0, 0.0, 0.0]} orientation={'horizontal'}>
+        <LinearLayout defaultItemAlignment={Alignment.topCenter} anchorPoint={AnchorPoint.topCenter} orientation={Orientation.vertical}>
```

Comment:
we don't need `orientation` here since vertical is the default.

Reply:
updated